.github/workflows/pr-custom-review.yml

```diff
@@ -0,0 +1,27 @@
+name: Check reviews
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - review_requested
+      - review_request_removed
+  pull_request_review:
+
+jobs:
+  pr-custom-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pr-custom-review
+        uses: paritytech/pr-custom-review@v1
+        with:
+          token: ${{ secrets.PRCR_TOKEN }}
+          # team that reviews changes in lines marked with 🔒
```

Comment:
This comment is incomplete since it's lines marked with `🔒` **and** lines below `🔒`. Any reason why you can't just link to the documentation here? https://github.com/paritytech/pr-custom-review#workflow-configuration

Reply:
Its not just a comment. It is kind of self protection measure for this repo and explicit demo of GHA in action as well. E.g. change of the team reviewing locked lines should be approved by the current team.
But yes, it could be placed in docs as well as real use case.

Reply:
> It is kind of self protection measure for this repo and explicit demo of GHA in action as well

I find that very difficult from infer from the comment. I suggest to use a more explicit message with the reasoning, like:

```suggestion
          # 🔒 PROTECTED: Changes to locks-review-team should be approved by the current locks review-team
```

Also this whole file [should be protected already](https://github.com/paritytech/pr-custom-review/blob/e52ae0a982c6b3c3f89336b2ceb5ca2e2b34bb35/src/constants.ts#L5) by [built-in-checks](https://github.com/paritytech/pr-custom-review/tree/e52ae0a982c6b3c3f89336b2ceb5ca2e2b34bb35#built-in-checks).